invenio_app_ils/records/loaders/schemas/json/series.py

```diff
@@ -57,6 +57,7 @@ class Meta:
     mode_of_issuance = fields.Str(required=True)
     note = fields.Str()
     pid = PersistentIdentifier()
+    publication_year = fields.Str()
```

Comment:
for the Document, we have this `required`. What about for Series? ping @kprzerwa